### ISSUE TITLE
ui5-date-picker: Text color contrast issues in High contrast black on white theme

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#2372]
**Describe the bug**
Text color contrast issues in High contrast black on white theme.

**To reproduce**
1. Go to https://github.wdf.sap.corp/pages/xweb/goalmanagementx/app.html?mock&sap-ui-theme=sap_belize_hcb#/goal-list
2. click 'Skip Tour' button
3. click more actions button on the top right of Goal Card
4. click 'Create Activity' button
5. select a new satus 'low'
6. open date picker for 'Achieved On'

**Actual Results**
The visual pink color ((#e269C9)) outline around the current date fails to meet 7:1 against the grey color background (#171717) and (#2E2E2E) of calendar widget.

**Expected behavior**
In high contrast themes, visual selected outline should meet 7:1 against their backgrounds. Hence modify the colors to meet this contrast ratio

**Screenshots**
![HCM - date pink outline issue](https://user-images.githubusercontent.com/13098801/96545605-49c95080-12db-11eb-9e21-bf28bbcda762.png)
![HCM - date pink outline issue against greyish shade](https://user-images.githubusercontent.com/13098801/96545613-4d5cd780-12db-11eb-8dc3-041135dd8667.png)

**Context**
- UI5 Web Components version：0.23.2
- OS/Platform: {...}
- Browser *(if relevant)*: {...}
- Other information: {...}

**Comment by ilhan007** [comment]
FYI @SAP/ui5-webcomponents-topic-b
